app/jenkins-for-jira-ui/src/components/StatusLabel/StatusLabel.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { cx } from '@emotion/css';
+import { statusLabel } from './StatusLabel.styles';
+
+export enum ConnectedState {
+	CONNECTED = 'CONNECTED',
+	DUPLICATE = 'DUPLICATE',
+	PENDING = 'PENDING'
+}
+
+type StatusLabelProps = {
+	text: string,
+	backgroundColor: string,
+	color: string
+};
+
+const StatusLabel = ({ text, backgroundColor, color }: StatusLabelProps): JSX.Element => {
```

Comment:
Created our own status label component. Checked out the Atlaskit one and the colours didn't jive with what's in our designs.